addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ZestZapRunner.java

```diff
@@ -388,13 +392,53 @@ public void handleResponse(ZestRequest request, ZestResponse response)
     public String handleClient(ZestScript script, ZestClient client)
             throws ZestClientFailException {
         try {
+            if (client instanceof ZestClientLaunch) {
+                String res = this.launchClient((ZestClientLaunch) client);
```

Comment:
New Java 17 handiness:

```suggestion
            if (client instanceof ZestClientLaunch client) {
                String res = this.launchClient(client);
```

Reply:
Oh! I didnt know about this, will def use it...